jena-base/src/main/java/org/apache/jena/atlas/lib/SetUtils.java

```diff
@@ -44,13 +42,26 @@ private SetUtils() {}
         return s1.stream().anyMatch(s2::contains) ;
     }
 
+    /**
+     * @param s1 a {@link Set}
+     * @param s2 a {@link Set}
+     * @return the eager set-theoretic union of {@code s1} and {@code s2}
+     * 
+     * @see org.apache.jena.ext.com.google.common.collect.Sets#union(Set, Set) for a lazy alternative
+     */
     public static <T> Set<T> union(Set<? extends T> s1, Set<? extends T> s2) {
         Set<T> s3 = new HashSet<>(s1) ;
         s3.addAll(s2) ;
         return s3 ;
     }
 
-    /** union difference intersection : those elements in s1 or s2 but not both. */
+    /**
+     * @param s1 a {@link Set}
+     * @param s2 a {@link Set}
+     * @return eager union difference intersection : those elements in {@code s1} or {@code s2} but not both
```

Comment:
"Eager" is my insertion, to contrast with the Guava one, but "union difference intersection" comes from the previous code-- I have no idea what that means! :)